data/human/human missions.txt

```diff
@@ -6849,3 +6849,318 @@ mission "FW Refugees to Humanika"
+			`	He hums to himself for a while before making some small talk. "You know, there's a part of me that would much rather be liberating slaves instead of filming them, but for now it seems like this is the best way for me to help. Say we did liberate them right now; there's just going to be another slave ship tomorrow. But if we create political and social will, that could stop the slave ships forever."`
+			choice
+				`	"I think you're right."`
+				`	"What about doing both?"`
+					goto plan
+
+			`	"And you know they'll use those poor people as human shields. Say we-" Bunker is cut off by the proximity alert.`
+				goto showtime
+
+			label plan
+			`	"Now you're talking! But it would take a lot of planning to do it without killing the people we are trying to liberate. You know they'll use those poor people as human shields. Say we-" Bunker is cut off by the proximity alert.`
+
+			label showtime
+			`	"Showtime!" he says, rubbing his hands together. Both of you watch as a capital ship accompanied by a small flotilla of Quicksilver escorts, all in pirate colors, land near the security detail, probably deafening them with the sound of their repulsors. As soon as the loading ramp is down, the security detail gets to work herding the slaves from one ship to another; hitting, shoving, and shouting at anyone that moves too slowly for their liking.`
+			`	"Pigs," blurts out Bunker. He zooms in on two men talking a few meters away from everyone else. One is in a suit, the other a black hooded robe. "Must be the leaders," he whispers as if they can hear him. Just then, the hooded man looks up to the sky as if he sees something. He motions, and the Quicksilvers fire up their engines.`
+			`	"Oh, I don't like the look of this," says Bunker.`
+			choice
+				`	"Keep filming."`
+				`	"Call back the drones immediately."`
+					goto "call drones"
+				`	"I think we'll have to leave the drones behind."`
+					goto "no can do"
+				`	"Did he see us?"`
+					goto "see us"
+
+			`	"I don't know. I think we oughta' get going," says Bunker, grimacing.`
+			choice
+				`	"That's an order!"`
+				`	"Okay. Call the drones back."`
+					goto "call drones"
+
+			`	"You're the captain," Bunker responds in a sore tone. Meanwhile, you continue to watch the display. It takes a couple minutes before the last of the slaves are loaded, followed by the security team. The two men in the suit and hood are the last to board their ships.`
+			`	You notice a slight hum from outside before getting raked with particle fire from the pirate Quicksilvers. You try to pull out to space but another pair of Quicksilvers appear, sending even more volleys of particle fire at you. The <ship>'s repulsors are hit and you feel your pilot seat go out underneath you as your ship falls from the sky, exploding on impact with <origin>'s surface.`
```

Comment:
```suggestion
			`	You barely notice the humming from outside just as the <ship> receives a hail of particle cannon fire from a pirate Quicksilver, appearing seemingly from nowhere as they whittle down the shield down rapidly. Quickly pushing your thrusters to the limit, you commanded your ship upwards, trying to make it to vacuum to get out of system before another pair of Quicksilvers send another volley of fire that broke through and hit your ship's repulsors. The last thing you experience is the ship slowing to a halt, levitating for just a moment before the pilot seat went out underneath you, the hull filled with screams as the <ship> makes a one way trip towards the surface.`
```

Reply:
Definitely not happy with my rewrite, but I wanted to make the death a lot more immersing.

Reply:
@TheMarksman-ES How about this?
```suggestion
			`	You barely notice the humming of thrusters from outside just as the <ship> receives a hail of particle cannon fire from a pirate Quicksilver, appearing seemingly from nowhere as they whittle down the shield down in a split second. You try to shunt your repulsors to full blast to get into orbit before another pair of Quicksilvers send another volley of fire that broke through and hit your ship's repulsors. The last thing you experience is the ship slowing to a halt, pausing in the air for just a moment before the pilot seat goes out underneath you, the cockpit filled with screams as the <ship> makes a one way trip towards the surface.`
```